client/src/index.css

```diff
@@ -1,13 +1,15 @@
+@import url('https://fonts.googleapis.com/css2?family=Cabin:ital,wght@0,400;0,500;0,600;0,700;1,400;1,500;1,600;1,700&family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap');
```

Comment:
Using Google fonts without stating it in the terms and conditions of the app violates GDPR because google font collects users' IP addresses for it to work. So I will say we avoid it?

https://cookie-script.com/blog/google-fonts-and-gdpr

Reply:
ok, thanks, i don't know any other way to get the font installed, but let me make some findings

Reply:
I suggested using Sora, Sans-serif as alternative